### PR TITLE
Added username and email, when user is signed in

### PR DESCRIPTION
**Pull request by SLiCK-303**

### CODE REVIEW

File: views/templates/hook/widget.tpl [hunk 1]
```diff
@@ -26,5 +26,11 @@
     s1.setAttribute('crossorigin', '*');
     s0.parentNode.insertBefore(s1, s0);
   })();
+  {if $cookie->email != ''}
+    $_Tawk_API.visitor = {
+      name  : '{$cookie->customer_firstname}',
```
Comment (firstred) [inline views/templates/hook/widget.tpl:31]: I always thought it was `$cookie->customer->firstname`. Shouldn't it be?

Also, could you escape for javascript please? Like so:
`name: '{$cookie->customer->firstname|escape:'javascript'}',`

**Review by firstred — COMMENTED** [review]
Looks like a great addition to me. Thanks!

I left a short review. Can you take care of the items, before I merge?

File: views/templates/hook/widget.tpl (hunk 1 above)
Comment (SLiCK-303) [reply views/templates/hook/widget.tpl:31]: I cant speak to the use of $cookie->customer->firstname, as I didnt test it that way.  The way I have it I run with, and it works fine.  I recommend you try it your way, and see if it works.....

Comment (firstred) [reply views/templates/hook/widget.tpl:31]: Ahh `$cookie->customer_firstname` is the only option that works

**Pull request merged**